gcpy/benchmark/modules/run_1yr_fullchem_benchmark.py

```diff
@@ -902,9 +913,9 @@ def gcc_vs_gcc_ops_budg(mon):
                 all_months_dev
             )[0]
 
-            # Plot models vs. observations (O3 for now)
-            mvo.make_benchmark_models_vs_obs_plots(
-                config["paths"]["obs_data_dir"],
+            # Plot models vs. EBAS O3 observations
```

Comment:
Is we had surface ozone from a different source would it be hard to change this?

Reply:
@lizziel: It depends on how different the data is from the data that we have.  I can change the name to to benchmark_model_vs_sfc_obs.py to make it more general, etc.

Reply:
@lizziel, As of commit 567342f, we now specify the model vs. obs label in the config file.  
```yaml
paths:
  main_dir: /n/holyscratch01/jacob_lab/ryantosca/BM/1yr
  results_dir: /n/holyscratch01/jacob_lab/ryantosca/BM/1yr/BenchmarkResults
  weights_dir: /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/GCHP/RegriddingWeights
  spcdb_dir: default
  #
  # Observational data dirs are on Harvard Cannon, edit if necessary
  #
  obs_data:
    ebas_o3:
      data_dir: /n/jacob_lab/Lab/obs_data_for_bmk/ebas_sfc_o3_2019
      data_label: "O3 (EBAS, 2019)"
    sondes:
      data_dir: /n/jacob_lab/Lab/obs_data_for_bmk/sondes_2010-2019
      data_file: allozonesondes_2010-2019.csv
      site_file: allozonesondes_site_elev.csv
```